build.sh

```diff
@@ -35,37 +37,35 @@ tcStatus() {
   tcMsg "buildStatus status='$status' text='$1'"
 }
 
-tcLog 'Show info'
-java -version
-mvn -version
-node --version
-npm --version
+tcLog "Show info (processors=$processors parallel=$parallel)"
+type java && java -version
+type mvn && mvn -version
+type node && node --version
+type npm && npm --version
+type pnpm && pnpm --version
 uname -a
 
 cmd="npm install --silent --quiet --no-progress"
 tcLog "Install NPM packages - $cmd"
 $cmd || exit 1
 
-cmd="scripts/mergeITs.js "`echo $elements`
-tcLog "Merge IT modules - $cmd"
-$cmd || exit 1
-
-cmd="mvn install -DskipTests -Drelease -B -T C$processors"
-tcLog "Installing flow components - $cmd"
-$cmd || exit 1
-
-if [ -n "$BUILD" ]
+cmd="mvn install -Drelease -B -q -T C$processors"
+tcLog "Unit-Testing and Installing flow components - $cmd"
+sleep 10
```

Comment:
Do we need to sleep here?

Reply:
In this case it seems that does not help, because for some reason this build fails frequently the first time.
But sleeping in the retry seems to help. Removing this.